pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_reconcile.go

```diff
@@ -52,20 +52,22 @@ import (
 )
 
 func (c *APIReconciler) reconcile(ctx context.Context, apiExport *apisv1alpha1.APIExport, apiDomainKey dynamiccontext.APIDomainKey) error {
+	logger := klog.FromContext(ctx)
+
 	if apiExport == nil || apiExport.Status.IdentityHash == "" {
 		c.mutex.RLock()
 		_, found := c.apiSets[apiDomainKey]
 		c.mutex.RUnlock()
 
 		if !found {
-			klog.V(3).Infof("No APIs found for API domain key %s", apiDomainKey)
+			logger.V(3).Info("No APIs found for API domain key")
```

Comment:
```suggestion
			logger.V(3).Info("no APIs found for API domain key")
```

(everywhere: we've been making lowercase)

Reply:
Next commit should have this fixing every file I touched in this PR. There's still a lot in the codebase that doesn't follow this convention that we should adjust in a follow up.